baselines/asset/test/gapic_asset_service_v1.ts.baseline

```diff
@@ -134,9 +134,18 @@ describe('v1.AssetServiceClient', () => {
         assert.strictEqual(result, fakeProjectId);
     });
 
+  const proxyquire = require('proxyquire');
+  const stub = sinon.stub().returns('deprecated');
+  const { AssetServiceClient} = proxyquire(
+    '../src/v1/asset_service_client.js',
+    {
+      'google-gax': {warn: stub},
+    }
+  );
```

Comment:
We need to take care the spaces here to align with up and down.
Ande we should declare the variable under scope of 
```
describe('{{ method.name.toCamelCase() }}', () => {
```
https://github.com/googleapis/gapic-generator-typescript/pull/918/files#diff-d3f0b12ce355b96d095da022dfd7d7cd666455cd78f423616c56a6609d3d8f50R251

Reply:
Thanks for reviewing! Quick question on the scope - right now as it's scope globally in the file, it invokes the stub for all of the tests. If the preference is to be scoped for each set of tests, then I can copy/paste the `proxyquire` code under each set of `describe('')` tests. Is the latter what you're suggesting?

Reply:
Okay holding off on this for a little bit as I chatted with Alex offline and we ideally do *not* want to use `proxyquire` as apparently it's a bit unwieldy to use. He's going to investigate how to do this and then I'll update here.

Reply:
Alright! Implemented using `stub` only by adding `warn` as part of the client constructor.